### PR TITLE
Modifies the README file to further clarify the steps needed to setup sync

### PR DESCRIPTION
**Pull request by Huji**
Based on my experience today setting up sync with help from AzaToth.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +74,8 @@ There is also a `deploy` command to deploy all Twinkle files live.
     ./sync.pl --deploy twinkle.js
     make deploy
 
+Note that for syncing to a custom wiki, you wil also need to specify the --lang and --family parameters too. For instance, to sync the files with `test.wmflabs.org` you should specify `--lang=test --family=wmflabs`. If you intend to use `make deploy` to deploy all the files at once, you may also need to modify the makefile to specify those parameters.
```
Comment (atlight) [inline README.md]: `wil` -> `will`

Comment (Huji) [inline README.md]: Okay I will fix it and try to slush the commits (hopefully I will do it correctly this time).

Comment (mc10) [inline README.md]: Easiest way is probably to make your changes and then amend the previous commit with `git commit --amend`, then force pushing it to overwrite the old commit.

Comment (Huji) [inline README.md]: Done!

**Comment by atlight** [comment]
Thanks!

**Pull request merged**